elasticsearch_metrics/field.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from elasticsearch_dsl import field as edsl_field
+
+__all__ = ["Date"]
+# Expose all fields from elasticsearch_dsl.field
+# We do this instead of 'from elasticsearch_dsl.field import *' because elasticsearch_metrics
+# has its own subclass of Date
+for each in (
+    field_name
+    for field_name in dir(edsl_field)
+    if field_name != "Date" and not field_name.startswith("_")
+):
+    field = getattr(edsl_field, each)
+    is_field_subclass = isinstance(field, type) and issubclass(field, edsl_field.Field)
```

Comment:
what is isinstance(field, type) for? What are you trying to exclude?

Reply:
This is `True` if `field` is a subclass of `elasticsearch_dsl.field.Field`. `isinstance(field, type)` ensures that `field` is a class. `issubclass` will error if its first argument is not a class, so that check needs to go first.